specifyweb/specify/views.py

```diff
@@ -5,29 +5,25 @@
 import json
 import mimetypes
 from functools import wraps
-from itertools import groupby
-from typing import Any, Callable, Dict, List, Optional, Union
+from typing import Any, Callable, Dict, List, Union
```

Comment:
(optional)
Looks like `Any`, `Dict`, and `List` types are never used in this file. 
Could they potentially be removed?